code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm

```diff
@@ -20,8 +20,7 @@
 	var/scrub_CO2 = TRUE
 	var/scrub_Toxins = FALSE
 	var/scrub_N2O = FALSE
-	var/scrub_BZ = FALSE
-	var/scrub_Freon = FALSE
+	var/scrub_Rare = FALSE
```

Comment:
Are you planning on keeping rare scrub? Having unique scrubs is nice QoL

Reply:
what is this

Reply:
Auroran yes, otherwise every time someone adds a new gas that means another variable AND another addition to the scrubber AI which would lead to huge bloat. This PR alone would add 5 or 6 buttons that way.

Reply:
Just turn the vars into an assoc lost and then iterate through it, you can pull all gas ids from meta_gas_info

Reply:
yeah scrubbers need a rewrite

Reply:
Agreed but I'm already having enough trouble having time for this PR, a scrubbers rework isn't really in the scope.

Reply:
this is fine for now imo
scrubbers can be fixed later

Reply:
Not a fan of this, but i guess can be done in seperate pr